client/src/actions/index.js

```diff
@@ -64,60 +77,62 @@ export const addPost = (newPost) => dispatch => {
             dispatch({ type: ADD_FAILURE, payload: err });
         })
 }
+>>>>>>> 253800119084e8f89f377b1660783d7130e29f37
```

Comment:
Function addPost is in here twice. Also these are merge marks showing the conflict has not finished resolving automatically. You can delete the merge marks and boil this down to one function, and that should work.

Reply:
Thank you, should be fixed now